.ci/ci.yml.tmpl

```diff
@@ -5,22 +5,22 @@ resource_types:
     - name: merged-downstreams
       type: docker-image
       source:
-          # Note: resource types cannot use credhub substitution - "nmckinley" is hardcoded here.
-          repository: nmckinley/merged-prs-resource
-          tag: v0.0.7
+          # Note: resource types cannot use credhub substitution - "gcr.io/magic-modules" is hardcoded here.
+          repository: gcr.io/magic-modules/merged-prs-resource
+          tag: 'latest'
```

Comment:
Probably best to stick to some sort of versioning scheme - we discussed `1.0.0`, congrats on the product launch.